libmproxy/models/connections.py

```diff
@@ -88,8 +88,10 @@ def finish(self):
 
 
 class ServerConnection(tcp.TCPClient, stateobject.StateObject):
-    def __init__(self, address):
-        tcp.TCPClient.__init__(self, address)
+    def __init__(self, address, source_address=None):
+        if source_address:
+            source_address = (source_address, 0)
```

Comment:
This is a bit confusing (i.e. inconsistent) to be honest. Can we just pass it through as-is?